docs/subnets/understanding-subnets.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Understanding Subnets"
+---
+
+# Understanding Subnets
+
+In Bittensor, a subnet is an incentive-based competition market that produces a specific kind of digital commodity related to artificial intelligence. It consists of a community of miners that produce the commodity, and a community of validators that measures the miners' work to ensure its quality.
+
+Rewards&mdash;emmissions of TAO (τ) from Bittensor&mdash;are distributed among miners and validators based on their performance within subnets, and based on the relative performance of subnets within Bittensor.
+
+## Competition, de-registration, and immunity period
+
+:::danger low-performing subnets risk deregistration
+:::
+
+Subnets must compete for weight within the Bittensor network. New subnets are created frequently, subnet performance is continuously monitored, and low-performance subnets are deregistered to make room for more successful ones. A subnet's performance is based on emissions earned, as well as other factors. See [Emissions](../emissions.md).
```

Comment:
A subnet's performance is (currently) based solely on the weights given to it by the root-set of Validators. 

This group evaluates subnets based on many criteria and sets the weight competitively/relative to the other subnets.

Reply:
We come to stake-weighted consensus over these weights, also following YC